### PR TITLE
if, ife and ifn snippets for Mustache

### PR DESCRIPTION
**Pull request by codeinabox**

### CODE REVIEW

File: snippets/mustache.snippets [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	{{/$1}}
+# {{#value}} ... {{/value}} {{^value}} ... {{/value}}
+snippet ife
+	{{#${1:value}}}
+	${2}
+	{{/$1}}
+	{{^$1}}
+	${3}
+	{{/$1}}
+
```
Comment (lpil) [inline snippets/mustache.snippets]: Remove this blank line please

**Comment by lpil** [comment]
Is there a reason to duplicate the snippets in the ultisnips format?

**Comment by codeinabox** [comment]
What's the convention? Should I just provide them in Snipmate format?

**Comment by lpil** [comment]
Ultisnips can read the snipmate format so there is no need to add ultisnips snippets unless it uses special ultisnips features :)

**Comment by codeinabox** [comment]
@lpil have removed that blank line and the duplication

**Comment by lpil** [comment]
Thanks!

**Pull request merged**